index.js

```diff
@@ -0,0 +1,118 @@
+// DOM ELEMENTS
+const likeButton = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+const form = document.querySelector("#new-goal-form")
+const doc = document.querySelector("body")
+doc.style.background = "AliceBlue"
+// console.log(form)
+
+// EVENT HANDLERS
+// made helper method for likes event listener. have to put fetch inside this method so it recognizes my newLikes variable. I'm learning!!
+const addLikes = () => {
+  let newLikes = parseInt(likes.textContent) + 1
+  likes.textContent = `${newLikes} Likes`
+
+  fetch("http://localhost:3000/players/1", {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    },
+    body: JSON.stringify({
+      likes: newLikes
+    })
+  })
+    .then(response => response.json())
+    .then(updatedPlayer => renderPlayer(updatedPlayer))
```

Comment:
This all looks good! You don't actually need to call `renderPlayer` here, since you're already updating the number of likes on the DOM on line 13:

```js
likes.textContent = `${newLikes} Likes`
```

Basically, you're doing both optimistic *and* pessimistic rendering, and you just need one or the other ✨